yorkie/src/androidTest/kotlin/dev/yorkie/document/json/JsonTreeTest.kt

```diff
@@ -1743,6 +1746,65 @@ class JsonTreeTest {
         }
     }
 
+    @Test
+    fun test_returning_range_from_index_correctly_withint_document_events() {
```

Comment:
```suggestion
    fun test_returning_range_from_index_correctly_within_document_events() {
```

Reply:
fixed in [34eea6e](https://github.com/yorkie-team/yorkie-android-sdk/pull/156/commits/34eea6eff226424abf9878b1a9e030855c68d08c).